### PR TITLE
54/global wildcard dns

### LINKED ISSUE #54: Use global wildcard DNS resolution for local domain names to always point to 127.0.0.1
Suggested by garrettw in Slack:

- Set up a new DNS name *.local.tokaido.io to auto-resolve to 127.0.0.1
- Update SSL certificate generator to create cert at *.local.tokaido.io
- Remove components that manipulate hosts file entries (except the ones that clean up with `tok destroy`

Thanks again Garrett!

### PR DESCRIPTION
**Pull request by otakumike**
fixes #54

### CODE REVIEW

File: system/hostsfile/main.go [hunk 1]
```diff
@@ -26,38 +17,6 @@ func confirmRemove(hostname string) bool {
...
-		if err := hosts.Flush(); err != nil {
-			return err
-		}
-
-		return nil
-	}
-
-	return nil
-}
-
```
Comment (yjimk) [inline system/hostsfile/main.go]: Really glad we're able to chop out the hostsfile stuff, will make it much easier on cross platform compat

File: system/ssl/main.go [hunk 2]
```diff
@@ -76,9 +81,48 @@ func FindOrCreateClientCert(certPath string) error {
...
+		return err
+	}
+	for _, name := range names {
+		err = os.RemoveAll(filepath.Join(dir, name))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```
Comment (yjimk) [inline system/ssl/main.go]: A function like this should live in the `fs` package IMO

File: system/ssl/trusted_windows.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ func AddTrustedCertToKeychain(certificate string) {
 
 // RemoveTrustedCertFromKeychain ...
 func RemoveTrustedCertFromKeychain(certificate string) {
-	fmt.Println("The generated SSL certificates need to be manually removed from your keychain if they were added during `tok up`. See https://tokaido.io/docs/config/#adding-a-trusted-certificate for more information.")
+	fmt.Println("    The generated SSL certificates need to be manually removed from your keychain if they were added during `tok up`. \n    See https://tokaido.io/docs/config/#adding-a-trusted-certificate for more information.")
```
Comment (yjimk) [inline system/ssl/trusted_windows.go]: If these certs messages are the same for each platform, could probably be a const in the `constants` package?

Comment (otakumike) [reply system/ssl/trusted_windows.go]: I thought about that, but they won't be long term. Once we have the new site up and running, I plan to add doc sections for updating SSL certs on each platform, so these will actually link to different places.

**Pull request merged**